### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by KangZhiDong**

### CODE REVIEW

**Comment by ShannonDing** [comment]
Plz move this PR to develop branch.

File: docs/en/Configuration_Client.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ export   NAMESRV_ADDR=192.168.0.1:9876;192.168.0.2:9876
 ```
 - HTTP static server addressing(default)
 
-After client started, it will access a http static server address, as: <http://jmenv.tbsite.net:8080/rocketmq/nsaddr>, this URL return the following contents:
+After client started, it will access an http static server address, as: <http://jmenv.tbsite.net:8080/rocketmq/nsaddr>, this URL return the following contents:
```
Comment (ShannonDing) [inline docs/en/Configuration_Client.md]: Why not use "an" here？

Comment (KangZhiDong) [reply docs/en/Configuration_Client.md]: an http

**Comment by duhenglucky** [comment]
@KangZhiDong please rebase first, this pr brought so many other commits.

**Comment by KangZhiDong** [comment]
> @KangZhiDong please rebase first, this pr brought so many other commits.

get .

**Pull request merged**